app/apollo/resolvers/group.js

```diff
@@ -351,6 +360,9 @@ const groupResolvers = {
         */
         groupsRbacSync( groups, { resync: false }, context ).catch(function(){/*ignore*/});
 
+        // Allow graphQL plugins to retrieve more information
+        context.pluginContext = {clusterNames: clusterIds, groupDetails: groupObjsToAdd};
```

Comment:
how can we get this to be
```
        context.pluginContext = { clusters: [], groups: [] };
```
Where `clusters` array members and groups array members are `{ uuid, name}` objects?  I _think_ the `groupObjsToAdd` object is like that, but having just cluster names or cluster ids is not sufficient.  And `clusterNames: clusterIds` also just confusing -- they're not _names_ :-).

Reply:
hmm, when I tested and ran the output for clusterIds, it did only give me the name I labeled it no id or uuid